.github/workflows/ci.yaml

```diff
@@ -28,8 +28,8 @@ jobs:
     - name: Check out code into the Go module directory
       uses: actions/checkout@v2
       with:
-        ref: ${{ github.event.pull_request.head.sha }}
         path: go/src/github.com/${{ github.repository }}
+        fetch-depth: 2
```

Comment:
A new "Last merge commit on the `GITHUB_REF` branch" will be generated for `GITHUB_SHA`, so we need to check out the repo with `fetch-depth: 2` so that we can get the real commit SHA when uploading the coverage report (to match another coverage report uploading by Jenkins CI)

Reply:
added a comment in https://github.com/pingcap/tidb-operator/pull/3725/commits/882915bb46a53be29af20878bb543f6bf6b41df2.

PTAL again